src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -4545,4 +4545,14 @@ public static void setSStableReadRatePersistenceEnabled(boolean enabled)
             conf.sstable_read_rate_persistence_enabled = enabled;
         }
     }
+
+    public static int getSAISegmentWriteBufferSpace()
+    {
+        return conf.sai_options.segment_write_buffer_space_mb;
+    }
+
+    public static void setSAISegmentWriteBufferSpace(int bufferSpace)
```

Comment:
If [CASSANDRA-15254](https://issues.apache.org/jira/browse/CASSANDRA-15254) completes soon enough, we might be able to avoid JMX and do it via a virtual table. This would probably be the preferred way, actually.

Reply:
I believe this method was left in got testing but is not currently used (or is ever likely to be). If it is needed then there are better ways of doing it for testing.